packages/react-internal/src/utilities/selection/SelectionZone.tsx

```diff
@@ -183,7 +187,7 @@ export class SelectionZone extends React.Component<ISelectionZoneProps, ISelecti
 
     return (
       <div
-        className={css('ms-SelectionZone', {
+        className={css('ms-SelectionZone', this.props.className ? this.props.className : '', {
```

Comment:
you don't need to ternary this; the `css` helper will ignore falsey values.

```suggestion
        className={css('ms-SelectionZone', this.props.className, {
```